### DISCUSSION TITLE
Dask: Loss of accuracy of XGBoost

### DISCUSSION THREAD

**Discussion by FelipeBDP** [dask/dask#7474] [Q&A]
When I use SKlearn to divide my dataset and after I use dd.from_pandas to convert Pandas Dataframe in Dask Dataframe, my accuracy is 50%. But when I divided my dataset using Sklearn and salve the new datasets, my accuracy is of 72%. 

When I use this code, I have 50% of accuracy:
```
async def main():
 async with Client('192.168.1.222:8786', asynchronous=True) as client:
    conjunto_de_dados = pd.read_csv('/pasta_compartilhada/dados/BaseReal/HIGGS-5milhoes-28att.csv')
    exemplos_ajustados = conjunto_de_dados.iloc[:,1:len(conjunto_de_dados.columns)]
    classificadores = conjunto_de_dados['classe']
    x_tr, y_tr,x_te, y_te = train_test_split(exemplos_ajustados, classificadores, train_size=0.85)
    x_tr= dd.from_pandas(x_tr,npartitions=100)
    x_te= dd.from_pandas(x_te,npartitions=100)
    y_tr= dd.from_pandas(y_tr,npartitions=100)
    y_test= dd.from_pandas(y_te,npartitions=100)
```

When I use this code, I have 72% of accuracy:
```
def processamento_dados():
    conjunto_de_dados = pd.read_csv('/pasta_compartilhada/dados/BaseReal/HIGGS-5milhoes-28att.csv')
    exemplos = conjunto_de_dados.iloc[:,1:len(conjunto_de_dados.columns)]
    exemplos_ajustados = conjunto_de_dados['classe']
    x_treinamento, y_treinamento,x_teste, y_teste = train_test_split(exemplos,  exemplos_ajustados, train_size=0.85)
    x_treinamento.to_csv('/pasta_compartilhada/dados/x_tr.csv', sep=',', index = False)
    y_treinamento.to_csv('/pasta_compartilhada/dados/y_tr.csv', sep=',', index = False)
    x_teste.to_csv('/pasta_compartilhada/dados/x_te.csv', sep=',',index = False)
    y_teste.to_csv('/pasta_compartilhada/dados/y_te.csv', sep=',', index = False)

async def main():
 async with Client('192.168.1.222:8786', asynchronous=True) as client:
    x_treina_read=dd.read_csv('/pasta_compartilhada/dados/x_te.csv')
    x_treina = x_treina_read.repartition(npartitions=100)
    y_treina_read=dd.read_csv('/pasta_compartilhada/dados/x_tr.csv')
    y_treina = y_treina_read.repartition(npartitions=100)
    x_testa_read=dd.read_csv('/pasta_compartilhada/dados/y_te.csv')
    x_testa = x_testa_read.repartition(npartitions=100)

```

About Dask ML, I yet don't succeed to install in Jetson TX2, and my tests need to be do in the Jetson TX2.
